SConstruct

```diff
@@ -778,6 +787,9 @@ elif env["CC"] == "cl": # Visual Studio
 elif "icc" in env.subst("$CC"):
     config.select("icc")
 
+elif "icx" in env.subst("$CC"):
+    config.select("icx")
+
```

Comment:
I know you didn't add this particular edge-case, but it occurs to me that if someone has the string `icx` somewhere in their path to the compiler, this case (or another one) may be incorrectly selected even if they didn't set `CC=icx`. I'm not sure how to best resolve this though, and it hasn't been a problem up to now, so maybe it doesn't need to be resolved.

Reply:
I agree, but I'd prefer to leave this as is, as this just follows what is done elsewhere.